example/grpc_example/server/main.go

```diff
@@ -0,0 +1,50 @@
+// Package main implements a server for Greeter service.
```

Comment:
update the README as well (and in the end update the docs)

Reply:
fixed